locales/en.json

```diff
@@ -59,6 +59,7 @@
   "cveDetailsPage.systemsWTCves": "Systems with this CVE",
   "cveHelper.MITRELink": "MITRE Database",
   "cveHelper.RHDBLink": "View in Red Hat CVE database",
+  "cveHelper.clusterLink": "Impacted Openshift clusters",
```

Comment:
I think fortmatJs should also allow to correctly spell the noun and distinguish between single/plurals. E.g. see the screenshot.. can we fix this?
![image](https://user-images.githubusercontent.com/31385370/214553643-f70cdf40-6661-4372-b8f5-530762f225af.png)

Reply:
By any chance do you know of where I can find an example to implement this? I've found some but they look overly complex

Reply:
Sure, here is one of the examples from ocp-advisor: https://github.com/RedHatInsights/ocp-advisor-frontend/blob/master/src/Messages.js#L228-L234